crates/fluvio-cli/src/client/consume/mod.rs

```diff
@@ -140,6 +140,12 @@ mod cmd {
         #[clap(short = 'b', long = "maxbytes", value_name = "integer")]
         pub max_bytes: Option<i32>,
 
+        /// Isolation level that consumer must respect.
+        /// Supported values: read_committed (ReadCommitted) - consume only committed records,
+        /// read_uncommitted (ReadUncommitted) - consume all records accepted by leader.
+        #[clap(long, value_parser=parse_isolation)]
+        pub isolation: Option<Isolation>,
```

Comment:
Not show stopper.  We should consolidate common configuration for consumer and producer instead of repeating